import-automation/import-progress-dashboard-frontend/client/package-lock.json

```diff
@@ -6821,9 +6821,9 @@
       }
     },
     "hosted-git-info": {
-      "version": "2.8.8",
-      "resolved": "https://registry.npmjs.org/hosted-git-info/-/hosted-git-info-2.8.8.tgz",
-      "integrity": "sha512-f/wzC2QaWBs7t9IYqB4T3sR1xviIViXJRJTWBlx2Gf3g0Xi5vI7Yy4koXQ1c9OYDGHN9sBy1DQ2AB8fqZBWhUg=="
+      "version": "2.8.9",
```

Comment:
Are changes in this file needed for subject tables import?

Reply:
This change came to effect when I pulled the upstream master into this branch.